app/models/taxon_presenter.js

```diff
@@ -1,5 +1,3 @@
-"use strict";
```

Comment:
Did you try running the prototype after removing this?

I get:

`SyntaxError: Block-scoped declarations (let, const, function, class)`